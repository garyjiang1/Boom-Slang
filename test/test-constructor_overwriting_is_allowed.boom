class MyClass:
	required:
		int x
	def construct(int x):
		println("i am overwriting the default constructor")

MyClass foo = MyClass(5)
