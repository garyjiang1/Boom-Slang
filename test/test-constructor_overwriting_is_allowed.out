i am overwriting the default constructor
